api/src/main/resources/workbench-api.yaml

```diff
@@ -5959,6 +5959,8 @@ definitions:
             description: >
               DEPRECATED - RW-8236. Remove after one cycle. Whether any modules have expired. Expiration information is also included with each
               module in the array; this field is provided for convenience.
+      demographicSurveyV2:
```

Comment:
should be identical to the changes in rdr-yaml.  If not, the RdrMapper will catch this.